src/tools/wasm-strip.cc

```diff
@@ -91,7 +99,13 @@ class BinaryReaderStrip : public BinaryReaderNop {
   Result BeginCustomSection(Index section_index,
                             Offset size,
                             std::string_view section_name) override {
-    if (sections_to_keep_.count(section_name) > 0) {
+    if (!sections_to_remove_.empty() &&
```

Comment:
I don't think you need the first half of this condition.  Just `if (sections_to_remove_.count(section_name) > 0) {` should be fine.

Reply:
Ah yeah... not sure what got me to put that there in the first place...